ingest/workflow/snakemake_rules/trigger_rebuild.smk

```diff
@@ -0,0 +1,21 @@
+"""
+This part of the workflow handles triggering new mpox builds after the
+latest metadata TSV and sequence FASTA files have been uploaded to S3.
+
+Designed to be used internally by the Nextstrain team with hard-coded paths
+to expected upload flag files.
+"""
+
+
+rule trigger_build:
+    message:
+        "Triggering dengue builds via repository action type `rebuild`."
```

Comment:
Since writing this workflow, we've decided as team to [avoid the `message` attribute in Snakemake rules](https://docs.nextstrain.org/en/latest/reference/snakemake-style-guide.html#avoid-the-message-rule-attribute).

Reply:
Fixed with https://github.com/nextstrain/dengue/pull/13/commits/a4d9ca74a43ac8bb3250f623d8272b76de9d281b and a separate PR to mkpx: https://github.com/nextstrain/monkeypox/pull/213

Later on, I expect this commit to disappear when I perform a rebase on mkpx ingest.